src/main/java/org/opensearch/neuralsearch/processor/factory/NormalizationProcessorFactory.java

```diff
@@ -53,9 +55,12 @@ public SearchPhaseResultsProcessor create(
         Map<String, Object> combinationClause = readOptionalMap(NormalizationProcessor.TYPE, tag, config, COMBINATION_CLAUSE);
 
         ScoreCombinationTechnique scoreCombinationTechnique = ScoreCombinationFactory.DEFAULT_METHOD;
+        Map<String, Object> combinationParams;
         if (Objects.nonNull(combinationClause)) {
-            String combinationTechnique = (String) combinationClause.getOrDefault(TECHNIQUE, "");
-            scoreCombinationTechnique = scoreCombinationFactory.createCombination(combinationTechnique);
+            String combinationTechnique = readOptionalStringProperty(NormalizationProcessor.TYPE, tag, combinationClause, TECHNIQUE);
```

Comment:
combinationTechnique is not optional but we use `readOptionalStringProperty`?

Reply:
it's optional, user may not set it and we need a default in this case, same for normalization

Reply:
I think `combinationClause` is optional but not `combinationTechnique`
`scoreCombinationFactory.createCombination` throws an exception when `combinationTechnique` is not set.

Reply:
I see, that's true we assign default technique in case the whole "clause" is missing. Let me add same for case when technique name is missing